CRM/NcnCiviZoom/Utils.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ *  NcnCiviZoom utils functions
+ *
+ * @package CiviCRM
+ */
+class CRM_NcnCiviZoom_Utils {
+
+	//Function to retrieve zoom settings
+	public static function getZoomSettings(){
+		$settings = CRM_Core_BAO_Setting::getItem(ZOOM_SETTINGS, 'zoom_settings');
+		return $settings;
+	}
+
+	public static function getCustomField(){
```

Comment:
Please fix this function. and can you please double check your code before commit.